interpreter/cli/cli.py

```diff
@@ -80,6 +80,18 @@
         "help_text": "optionally enable safety mechanisms like code scanning; valid options are off, ask, and auto",
         "type": str,
         "choices": ["off", "ask", "auto"]
+    },
+    {
+        "name": "threads",
```

Comment:
This argument never actually gets passed to the `interpreter` instance because `interpreter.threads` doesn't exist.

In [`core.py`](https://github.com/KillianLucas/open-interpreter/pull/595/files#diff-44ef24ada573d406ea9f66d0669046a162d58d6db6bbd83308f4704b7eedb5f7R35) the `Interpreter` Class attribute is called `num_threads`, so the argument parsing logic drops the `--threads` passed to the CLI because it doesn't find a corresponding attribute.